mkdocs/tests/integration.py

```diff
@@ -35,17 +35,20 @@
               required=True)
 def main(output=None):
 
+    print("Building themes.")
```

Comment:
These print statements are not getting printed inline with the log statements. Instead they are all groupeds together at the end. These should be writing to the log rather than print statements. See the log [L1146-L1164](https://ci.appveyor.com/project/d0ugal/mkdocs/build/2.812/job/sy52anb8i5vtpo0r#L1146)